### ISSUE TITLE
How to Invoke?

### ISSUE THREAD

**Issue by jerryxm23** [ericsonj/vscode-verilogformar#3]
How do you invoke the formatter from within VS Code? It's not obvious from the README file.

**Comment by jdawgzim** [comment]
Yeah, how do we use this in VS Code?  Is it only usable in the console?  Does the VS Code extension do anything?  

I've been wanting something like VHDL Beautify for a long time.  Getting this for Verilog in other editors would be amazing!

**Comment by Sciroccogti** [comment]
I use this by `Shift+Alt+F`, which is the default shortcut for formating codes in VScode.

**Comment by lwshowl** [comment]
me to ,I have definitely set the path right , I can use that in PowerShell and it works , but right click format or using the hot key never works for me
